openrtb_ext/request_wrapper.go

```diff
@@ -333,6 +337,13 @@ func (ue *UserExt) unmarshal(extJson json.RawMessage) error {
 		}
 	}
 
+	if consentedProviderSettingsJson, hasCPSettings := ue.ext["ConsentedProvidersSettings"]; hasCPSettings {
```

Comment:
Please extract this value to constant - you use it in many places. Also should it start from lowercase?

Reply:
I'm not sure what value to put into a constant. `consentedProviderSettingsJson`? Inside `(ue *UserExt) unmarshal(extJson json.RawMessage) error` it is only used once.

Reply:
Oh I meant string constant `"ConsentedProvidersSettings"`

Reply:
Has this string been extracted?

Reply:
Has not, corrected